src/server/game/Battlegrounds/Zones/BattlegroundAB.cpp

```diff
@@ -183,6 +186,16 @@ void BattlegroundAB::PostUpdateImpl(uint32 diff)
             EndBattleground(ALLIANCE);
         else if (m_TeamScores[TEAM_HORDE] >= BG_AB_MAX_TEAM_SCORE)
             EndBattleground(HORDE);
+
+        /// @workaround The original AreaTrigger is covered by a bigger one and not triggered on client side.
+        for (BattlegroundPlayerMap::const_iterator itr = GetPlayers().begin(); itr != GetPlayers().end(); ++itr)
```

Comment:
Yeaaaaah, don't do this on every update like this

Reply:
Hmmm.. What would you suggest as an alternative?

Reply:
Is this something we could do in ```UpdatePositionData``` ? Could add all related area triggers there.